blockchain.go

```diff
@@ -1037,7 +1040,13 @@ func (b *Blockchain) GetAccountStorage(address sdk.Address) (*AccountStorage, er
 		WithMaxKeySizeAllowed(b.vmCtx.MaxStateKeySize).
 		WithMaxValueSizeAllowed(b.vmCtx.MaxStateValueSize)
 
-	txnPrograms := programs.NewEmptyPrograms()
+	derivedBlockData := derived.NewEmptyDerivedBlockData()
+	derivedTxnData, err := derivedBlockData.NewSnapshotReadDerivedTransactionData(
+		derived.EndOfBlockExecutionTime,
+		derived.EndOfBlockExecutionTime)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
nit, maybe some method like `derived.NewEmptyDerivedTransactionData()` in the future to fvm?

Reply:
you're using an internal api here.  if you're using the fvm public interface, these are optional

Reply:
hmm I didn't know you can do this via fvm public interfaces. let me have a look and make a PR, thanks @pattyshack

Reply:
i don't think there's an equivalent public interface.  i'm just pointing out that fvm is designed around the VM (Run) interface in general

Reply:
ah ok got it thanks, though probably I still workaround fvm here.